BUILD

```diff
@@ -144,6 +150,25 @@ The dinit executable will be put in /sbin (or rather, in $DESTDIR/sbin), which m
 path for normal users. Consider making a symbolic link to /usr/sbin/dinit.
 
 
+Special notes for "configure" script
```

Comment:
Put these in a separate file, or in the `--help` output from the configure script.